examples/svelte-apollo/src/main.js

```diff
@@ -0,0 +1,7 @@
+import App from './App.svelte';
+
+const app = new App({
+	target: document.body,
```

Comment:
strange indentation?

Reply:
at line 4?

Reply:
yes. Seem to be 4 spaces instead of 2 spaces?

Reply:
It more like enter done after the brackets, no spaces.

Reply:
Seems like it's a tab instead of 2 spaces in line 4.

Reply:
Fixed